Add 'on behalf of' fields to car loan

ffossum opened the pull request:
Valgte å legge til en boolean i tillegg til en optional record type slik at mottaker kan være sikker på at dette er noe brukeren har tatt forhold til da de fylte ut skjema.

ffossum committed:
Add 'on behalf of' fields to car loan

ffossum commented:
@aspic @amumurst har dere vært borti feilene på travis før?

amumurst commented:
Har ikke vært travis her før, jeg som enablet det for noe annet her også ble denne dratt med ser det ut til

mehlium commented on src/main/avro/billan.avsc:
```diff
@@ -141,6 +141,35 @@
+          "type": "record",
+          "fields": [
+            {
+              "name": "navn",
```
Tydeleggjere at det er fullt namn? :thinking:

ffossum committed:
make paaVegneAvAnnen optional

for old applications the field can then be null to indicate user hasn't entered a value

ffossum committed:
fix travis build

https://travis-ci.community/t/solved-oraclejdk8-installation-failing-still-again/3428

ffossum committed:
update billan test file

ffossum committed:
rename navn -> fulltNavn
